res/sections/45-WirelessStandards.tex

```diff
@@ -0,0 +1,348 @@
+  traverse
+\end{itemize}
+
+\subsubsection{Urban multi-hop broadcasting protocols} This broadcasting
+protocols uses \textbf{jamming} signals to determine the next forwarder:
+vehicles receiving an alert message emit a jamming signal for a time that's
+proportional to the distance from the sender. The last one stops the jamming
+signal and forward the message.
+
+The problem of this solution is that is slow, and for this is not suitable for
+alert messages (jamming signal phase delays the transmission of the message).
+
+\subsubsection{Fast Broadcast}
+Fast broadcast is designed to have Alert Messages covering the area of interest
+in less time possible. It uses two type of packets: \textit{Hello messages} and
+\textit{Alert messages}. Hello messages contain the sender's position and the
+maximum forward distance from which another vehicle has been heard transmitting
+another hello message. In particular it contains:
+\begin{itemize}
+\item CMBR (\textit{Current Maximum Backward Range}): the maximum backward
+  distance at which some car would be able to hear me
+\item CMFR (\textit{Current Frontward Range}): the maximum distance from which
+  I have been able to hear another car in front of me
+\end{itemize}
+
+Fast Broadcast has two phases:
+\begin{enumerate}
+\item estimation of transmission range between two vehicles (called
+  \textit{hello messages}). This phase has little overhead and it's divided in
+  rounds: at every round
+  hello messages are randomly sent
+\item transmission is used during broadcasting to reduce message number of hops
+  (\textit{alert messages}\footnote{This message includes the estimated
+  transmission range for that hop.})
+\end{enumerate}
+
+\begin{figure}
+  \centering
+  \includegraphics[scale=0.35]{fastBroadcast}
+  \caption[Hello Message propagation]{A little illustration of how hello
+    messages are delivered through the cars.}
+  \label{fig:802.11ws:helloMessageDelivery}
+\end{figure}
+
+When a car receives a message, as illustrated in
+Figure~\ref{fig:802.11ws:helloMessageDelivery}, it can be from two directions:
+in front of it (and in that case it's an update), or behind it (and in that case
+you update your transmission range).
+
+The characteristics of this multi-hop broadcast protocol for vehicular networks
+are:
+\begin{itemize}
+\item estimation of the maximum transmission range with few hello messages
+\item minimization of the number of hops to be traversed
+\item reduction of the delivery time
+\end{itemize}
+
+\paragraph*{Handling an AV} When an Abnormal Vehicle (AV) occurs, an alert
+message is generated, and received by the nearby cars. The cars that received
+the message wait a time that is proportional to the node's position with
+respect to the estimated maximum transmission range. In particular, farthest
+nodes transmit before the others: if another car that is farther from the source
+has already forwarded the alter message, then the considered car abort its
```

Comment:
... then the others car abort its...

From paper:

If, while waiting, some of the following cars already transmitted the message, preceding ones abort their forwarding countdowns as the message has already been propagated.